test/core/test_repositories_integration.py

```diff
@@ -6,42 +6,53 @@
 """
 
 import datetime
+import time
 from unittest import mock
 from unittest.mock import Mock
 
 import pytest
+from db.data_models import Base, Instrument, Job, JobOwner, JobType, Run, Script, State
 
-from fia_api.core.model import Base, Instrument, Reduction, ReductionState, Run, Script
 from fia_api.core.repositories import ENGINE, SESSION, Repo, test_connection
-from fia_api.core.specifications.reduction import ReductionSpecification
-
-# pylint: disable = redefined-outer-name
+from fia_api.core.specifications.job import JobSpecification
 
+TEST_JOB_OWNER = JobOwner(experiment_number=1)
+TEST_JOB_OWNER_2 = JobOwner(experiment_number=2)
+TEST_INSTRUMENT_1 = Instrument(instrument_name="instrument 1", latest_run=1, specification={"foo": "bar"})
+TEST_INSTRUMENT_2 = Instrument(instrument_name="instrument 2", latest_run=1, specification={"foo": "bar"})
 TEST_SCRIPT = Script(script="print('Script 1')", sha="some_sha", script_hash="some_hash")
-TEST_REDUCTION = Reduction(
-    reduction_start=datetime.datetime.now(datetime.UTC),
-    reduction_state=ReductionState.NOT_STARTED,
-    reduction_inputs={"input": "value"},
+TEST_JOB = Job(
+    start=datetime.datetime.now(datetime.UTC),
+    owner=TEST_JOB_OWNER_2,
+    state=State.NOT_STARTED,
+    inputs={"input": "value"},
     script=TEST_SCRIPT,
+    instrument=TEST_INSTRUMENT_1,
+    job_type=JobType.AUTOREDUCTION,
 )
-TEST_REDUCTION_2 = Reduction(
-    reduction_start=datetime.datetime.now(datetime.UTC),
-    reduction_state=ReductionState.UNSUCCESSFUL,
-    reduction_inputs={"input": "value"},
+time.sleep(1)
+TEST_JOB_2 = Job(
+    start=datetime.datetime.now(datetime.UTC),
+    owner=TEST_JOB_OWNER,
+    state=State.UNSUCCESSFUL,
+    inputs={"input": "value"},
     script=TEST_SCRIPT,
+    instrument=TEST_INSTRUMENT_1,
+    job_type=JobType.AUTOREDUCTION,
 )
-TEST_REDUCTION_3 = Reduction(
-    reduction_start=datetime.datetime.now(datetime.UTC),
-    reduction_state=ReductionState.SUCCESSFUL,
-    reduction_inputs={"input": "value"},
+
+TEST_JOB_4 = Job(
```

Comment:
It's confusing that it goes from TEST_JOB_2 to TEST_JOB_4

Reply:
It is, looks like we deleted no.3